db/migrate/20160108192239_devise_create_admins.rb

```diff
@@ -0,0 +1,18 @@
+class DeviseCreateAdmins < ActiveRecord::Migration
+  def self.up
+    create_table(:admins) do |t|
+      t.string :email,              :null => false, :default => ""
+      t.string :encrypted_password, :null => false, :default => ""
+      t.integer  :sign_in_count, :default => 0
```

Comment:
Use the new Ruby 1.9 hash syntax.<br>Put one space between the method name and the first argument.